ui/imports/shared/views/chat/ChatContextMenuView.qml

```diff
@@ -208,6 +218,18 @@ StatusMenu {
         enabled: !root.isCommunityChat || root.amIChatAdmin
     }
 
+    StatusAction {
+        id: clearHistoryMenuItem
+        objectName: "clearHistoryMenuItem"
+        enabled: (root.chatType === Constants.chatType.oneToOne)
+        text: qsTr("Clear History")
+        icon.name: "delete"
+        type: StatusAction.Type.Danger
+        onTriggered: {
+            Global.openPopup(clearChatConfirmationDialogComponent);
+        }
+    }
```

Comment:
This one seems duplicated from the one above. One is for OneToOne and the other for other types of chat, but I don't see any other differences. Is it just about the positioning?

Reply:
Exactly, and I see no other way around it

Reply:
I looked at the designs and it is my humble opinion that the "close chat" and "leave group" should both be at the bottom.
It makes it more standardized and also makes it so we could avoid the duplication :smile: 
I'll let @benjthayer decide

Reply:
Yeah, so I didn't change that one from the original PR... at the same time, I wasn't able to find out the difference between a one-to-one chat and otehr chat types